src/model.rs

```diff
@@ -83,10 +78,10 @@ impl Pod {
         output_stream_map: BTreeMap<String, StreamInfo>,
         recommended_cpus: f32,
         recommended_memory: u64,
+        annotation: Option<Annotation>,
```

Comment:
Similar to above, I feel like when defining the pod, the name description and version comes to mind first follow by the rest of the spec?

Reply:
See above comment. This would be a good argument for pushing it higher and having user flow > putting optional args at the end.